redaxo/src/addons/be_style/plugins/customizer/lang/de_de.lang

```diff
@@ -10,8 +10,10 @@ customizer_codemirror_langs = Zusätzliche Highlighter
 customizer_codemirror_langs_text = (markdown, textile, gfm, yaml, sql ...)
 customizer_codemirror_tools = Zusätzliche Features
 customizer_codemirror_tools_text = (Code-Folding, Bracket-Matching, Tag-Matching, Close-Brackets, Search)
+customizer_codemirror_autoresize = AutoResize
+customizer_codemirror_autoresize_text = (CodeMirror passt sich in der Höhe an den Inhalt an)
 
-customizer_codemirror_info = Sobald CodeMirror aktiviert ist, wird dieser bei Templates, Modulen, Aktionen und bei Cronjobs automatisch verwendet. Sofern man selbst den CodeMirror verwenden möchte, bitte folgende Form verwenden:<br><br><code><textarea class="codemirror" data-codemirror-theme="themename" data-codemirror-mode="php/htmlmixed"></textarea></code><br><br>Die Parameter <i>data-codemirror-theme</i> und <i>data-codemirror-mode</i> sind optional.
+customizer_codemirror_info = Sobald CodeMirror aktiviert ist, wird dieser bei Templates, Modulen, Aktionen und bei Cronjobs automatisch verwendet. Sofern man selbst den CodeMirror verwenden möchte, bitte folgende Form verwenden:<br><br><code><textarea class="codemirror" data-codemirror-theme="themename" data-codemirror-mode="php/htmlmixed"></textarea></code><br><br>Die Parameter <i>data-codemirror-theme</i> und <i>data-codemirror-mode</i> sind optional.<br><br>Mit den Tasten F11 und ESC kann der Fullscreen-Modus aufgerufen werden.
```

Comment:
als kleiner Hinweis, wenn du bestehende Übersetzungen bearbeitest, bitte bei den anderen Sprachen die Übersetzungseinträge entfernen, damit diese von unseren Experten über traduko neu befüllt werden können.

(Gregor korrigiere mich, wenn ich falsch liege)

Reply:
Okay, werde ich in Zukunft berücksichtigen :)

Reply:
@aeberhard wenn du die `customizer_codemirror_info` für die anderen Sprachen entfernst passt das für mich. Alternativ kann ich das auch für dich machen, musst nur sagen :wink:

Reply:
Habe ich angepasst @bloep :)